samza-core/src/main/java/org/apache/samza/table/remote/CreditFunction.java

```diff
@@ -32,5 +31,13 @@
  * @param <K> the type of the key
  * @param <V> the type of the value
  */
-public interface CreditFunction<K, V> extends Function<KV<K, V>, Integer>, Serializable {
+@InterfaceStability.Unstable
+public interface CreditFunction<K, V> extends Serializable {
```

Comment:
Can this be an inner class within the Throttler, didn't feel it's worth exposing?